src/App.vue

```diff
@@ -39,6 +43,12 @@ export default {
       localStorage.setItem("todos", JSON.stringify(todosLocalStorage));
       this.todos.splice(key, 1);
     },
+    setCompleteTaks: function(key) {
+      let todosLocalStorage = JSON.parse(localStorage.getItem("todos"));
+      todosLocalStorage.splice(key, 1, "completed");
+      localStorage.setItem("todos", JSON.stringify(todosLocalStorage));
+      this.todos.splice(key, 1, "completed");
```

Comment:
Hier wird genau das gleiche gemacht wie in Zeile 48 nur auf das interne todos Array.

Im Prinzip musst du jeweils im todosLocalStorage-Array und im todos-Array das sub Array ansprechen und dort "uncompleted" auf "completed" setzen.

So würde das beim todosLocalStorage-Array erreicht werden:
`todosLocalStorage[key][1] = "completed";`